pkg/controller/controller_binding.go

```diff
@@ -86,7 +88,7 @@ func isServiceInstanceCredentialFailed(binding *v1alpha1.ServiceInstanceCredenti
 
 // an error is returned to indicate that the binding has not been
 // fully processed and should be resubmitted at a later time.
```

Comment:
Can you make this a proper godoc comment about the method and document what the boolean return value represents?